src/lineup/internal/LineUpPanelActions.ts

```diff
@@ -114,14 +128,20 @@ export default class LineUpPanelActions extends EventHandler {
 
   set collapse(value: boolean) {
     this.node.classList.toggle('collapsed', value);
+    if (value && this.options.enableSidePanel !== 'top') {
+      this.removeTabButtonHighlighting();
+    }
```

Comment:
Why is this necessary for a side panel that is not top? Please add some comment to the code.